### DISCUSSION TITLE
Connect redash to aws athena over private endpoint

### DISCUSSION THREAD

**Discussion by ashwinkhushlani** [getredash/redash#6712] [Q&A]
Want to connect Redash with AWS Athena using Athena's Private Endpoint. Is there a way to achieve the same?

**Comment by ognjen-it** [comment]
Did you try it?
I'm not sure why you cannot do that since redash is server side app.

**Reply by ashwinkhushlani** [reply]
yes, There is no option to add a private endpoint url.

**Reply by ognjen-it** [reply]
I would suggested you to try add next variable to all containers with private endpoint as value:
AWS_ENDPOINT_URL_ATHENA

Since redash using boto3 and pyathena, I hope it will work.
